### DISCUSSION TITLE
Y Axis Not Working.

### DISCUSSION THREAD

**Discussion by davolker1** [arkypita/LaserGRBL#834] [Q&A]
The X Axis on the Laser is working but I can't get the Y Axis on my Engraver to work. Any Ideas On how to troubleshoot it. Checked connections and they seem fine. I don't know if it is a software setting that needs to be checked.  It is one I ordered off of Amazon. 

https://www.amazon.com/gp/product/B082WRFP6F/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1

Thanks

**Comment by StuartB4** [comment]
There's not really anything in LaserGRBL that would stop the Y axis working. But you can check the GRBL settings by starting LaserGRBL and click Connect. Then goto Grbl menu and Grbl Configuration.
Check the 4 items I've highlighted to make sure they are not set to 0.
Do the Y motors make any noise at all, like they are trying to move.
If not you could have a faulty stepper driver board, you could try putting the X axis board  in the Y axis socket to see if they work. If they do then you will have a faulty Y axis stepper driver board.

![gfgffgf](https://user-images.githubusercontent.com/16980026/72009699-539f2d00-324e-11ea-9bba-0839f2b53cd5.jpg)

The other thing is you may need to adjust the stepper driver current (Vref voltage) on the little red board that the Y axis motors are connected to on the main controller board. You would need a multimeter to adjust them correctly. They are probably set in the factory but sometimes they need a bit of tweaking. There's a video to show you how to do it. You dont need the special power supply or the driver board that they use in the video. You just DISCONNECT the Y axis motors from the controller board, connect the Black lead of the multimeter to a ground point on your machine's power supply and the Red lead to the screwdriver or the top of the adjuster pot. 
Switch your machine on.
Switch the multimeter to the 2 volt DC range and gradually adjust the pot until you get the correct voltage reading, which will be in millivolts. Something like 0.5 Volts or 500 millivolts. This varies depending on the stepper driver and the motors. If you set it at say 300 millivolts and the motors struggle you just increase the current a bit more, sometimes it's just trying different settings until it works smoothly.
Here's the video. https://www.pololu.com/blog/484/video-setting-the-current-limit-on-pololu-stepper-motor-driver-carriers
